2024-10-06 21:14:21,293 INFO    StreamThr :7667 [internal.py:wandb_internal():85] W&B internal server running at pid: 7667, started at: 2024-10-06 21:14:21.292552
2024-10-06 21:14:21,293 DEBUG   HandlerThread:7667 [handler.py:handle_request():158] handle_request: status
2024-10-06 21:14:21,294 INFO    WriterThread:7667 [datastore.py:open_for_write():87] open: /Users/milind/Desktop/valence/milind_code/wandb/run-20241006_211421-es5i1noe/run-es5i1noe.wandb
2024-10-06 21:14:21,296 DEBUG   SenderThread:7667 [sender.py:send():379] send: header
2024-10-06 21:14:21,296 DEBUG   SenderThread:7667 [sender.py:send():379] send: run
2024-10-06 21:14:21,504 INFO    SenderThread:7667 [dir_watcher.py:__init__():211] watching files in: /Users/milind/Desktop/valence/milind_code/wandb/run-20241006_211421-es5i1noe/files
2024-10-06 21:14:21,504 INFO    SenderThread:7667 [sender.py:_start_run_threads():1188] run started: es5i1noe with start time 1728267261.292987
2024-10-06 21:14:21,510 DEBUG   HandlerThread:7667 [handler.py:handle_request():158] handle_request: check_version
2024-10-06 21:14:21,510 DEBUG   SenderThread:7667 [sender.py:send_request():406] send_request: check_version
2024-10-06 21:14:21,563 DEBUG   HandlerThread:7667 [handler.py:handle_request():158] handle_request: run_start
2024-10-06 21:14:21,571 DEBUG   HandlerThread:7667 [system_info.py:__init__():26] System info init
2024-10-06 21:14:21,571 DEBUG   HandlerThread:7667 [system_info.py:__init__():41] System info init done
2024-10-06 21:14:21,571 INFO    HandlerThread:7667 [system_monitor.py:start():194] Starting system monitor
2024-10-06 21:14:21,571 INFO    SystemMonitor:7667 [system_monitor.py:_start():158] Starting system asset monitoring threads
2024-10-06 21:14:21,571 INFO    HandlerThread:7667 [system_monitor.py:probe():214] Collecting system info
2024-10-06 21:14:21,573 INFO    SystemMonitor:7667 [interfaces.py:start():188] Started cpu monitoring
2024-10-06 21:14:21,575 INFO    SystemMonitor:7667 [interfaces.py:start():188] Started disk monitoring
2024-10-06 21:14:21,575 ERROR   HandlerThread:7667 [gpu_apple.py:probe():176] GPU stats error: [Errno 2] No such file or directory: '/opt/anaconda3/lib/python3.12/site-packages/wandb/bin/apple_gpu_stats'
Traceback (most recent call last):
  File "/opt/anaconda3/lib/python3.12/site-packages/wandb/sdk/internal/system/assets/gpu_apple.py", line 164, in probe
    subprocess.check_output(command, universal_newlines=True)
  File "/opt/anaconda3/lib/python3.12/subprocess.py", line 466, in check_output
    return run(*popenargs, stdout=PIPE, timeout=timeout, check=True,
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/opt/anaconda3/lib/python3.12/subprocess.py", line 548, in run
    with Popen(*popenargs, **kwargs) as process:
         ^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/opt/anaconda3/lib/python3.12/subprocess.py", line 1026, in __init__
    self._execute_child(args, executable, preexec_fn, close_fds,
  File "/opt/anaconda3/lib/python3.12/subprocess.py", line 1953, in _execute_child
    raise child_exception_type(errno_num, err_msg, err_filename)
FileNotFoundError: [Errno 2] No such file or directory: '/opt/anaconda3/lib/python3.12/site-packages/wandb/bin/apple_gpu_stats'
2024-10-06 21:14:21,576 INFO    SystemMonitor:7667 [interfaces.py:start():188] Started gpuapple monitoring
2024-10-06 21:14:21,577 ERROR   gpuapple  :7667 [gpu_apple.py:sample():110] GPU stats error: [Errno 2] No such file or directory: '/opt/anaconda3/lib/python3.12/site-packages/wandb/bin/apple_gpu_stats'
Traceback (most recent call last):
  File "/opt/anaconda3/lib/python3.12/site-packages/wandb/sdk/internal/system/assets/gpu_apple.py", line 64, in sample
    subprocess.check_output(command, universal_newlines=True)
  File "/opt/anaconda3/lib/python3.12/subprocess.py", line 466, in check_output
    return run(*popenargs, stdout=PIPE, timeout=timeout, check=True,
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/opt/anaconda3/lib/python3.12/subprocess.py", line 548, in run
    with Popen(*popenargs, **kwargs) as process:
         ^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/opt/anaconda3/lib/python3.12/subprocess.py", line 1026, in __init__
    self._execute_child(args, executable, preexec_fn, close_fds,
  File "/opt/anaconda3/lib/python3.12/subprocess.py", line 1953, in _execute_child
    raise child_exception_type(errno_num, err_msg, err_filename)
FileNotFoundError: [Errno 2] No such file or directory: '/opt/anaconda3/lib/python3.12/site-packages/wandb/bin/apple_gpu_stats'
2024-10-06 21:14:21,579 DEBUG   HandlerThread:7667 [system_info.py:probe():152] Probing system
2024-10-06 21:14:21,580 INFO    SystemMonitor:7667 [interfaces.py:start():188] Started memory monitoring
2024-10-06 21:14:21,580 INFO    SystemMonitor:7667 [interfaces.py:start():188] Started network monitoring
2024-10-06 21:14:21,583 DEBUG   HandlerThread:7667 [gitlib.py:_init_repo():56] git repository is invalid
2024-10-06 21:14:21,583 DEBUG   HandlerThread:7667 [system_info.py:probe():200] Probing system done
2024-10-06 21:14:21,583 DEBUG   HandlerThread:7667 [system_monitor.py:probe():223] {'os': 'macOS-14.6.1-arm64-arm-64bit', 'python': '3.12.2', 'heartbeatAt': '2024-10-07T02:14:21.580426', 'startedAt': '2024-10-07T02:14:21.285250', 'docker': None, 'cuda': None, 'args': ('data.pkl',), 'state': 'running', 'program': '/Users/milind/Desktop/valence/milind_code/non_llm.py', 'codePathLocal': 'non_llm.py', 'codePath': 'non_llm.py', 'host': 'Milinds-MacBook-Pro.local', 'username': 'milind', 'executable': '/opt/anaconda3/bin/python', 'cpu_count': 8, 'cpu_count_logical': 8, 'disk': {'/': {'total': 228.27386474609375, 'used': 166.39567947387695}}, 'gpuapple': {'type': 'arm', 'vendor': 'Apple'}, 'memory': {'total': 8.0}}
2024-10-06 21:14:21,583 INFO    HandlerThread:7667 [system_monitor.py:probe():224] Finished collecting system info
2024-10-06 21:14:21,583 INFO    HandlerThread:7667 [system_monitor.py:probe():227] Publishing system info
2024-10-06 21:14:21,583 DEBUG   HandlerThread:7667 [system_info.py:_save_conda():209] Saving list of conda packages installed into the current environment
2024-10-06 21:14:22,509 INFO    Thread-12 :7667 [dir_watcher.py:_on_file_created():271] file/dir created: /Users/milind/Desktop/valence/milind_code/wandb/run-20241006_211421-es5i1noe/files/conda-environment.yaml
2024-10-06 21:14:23,588 ERROR   gpuapple  :7667 [gpu_apple.py:sample():110] GPU stats error: [Errno 2] No such file or directory: '/opt/anaconda3/lib/python3.12/site-packages/wandb/bin/apple_gpu_stats'
Traceback (most recent call last):
  File "/opt/anaconda3/lib/python3.12/site-packages/wandb/sdk/internal/system/assets/gpu_apple.py", line 64, in sample
    subprocess.check_output(command, universal_newlines=True)
  File "/opt/anaconda3/lib/python3.12/subprocess.py", line 466, in check_output
    return run(*popenargs, stdout=PIPE, timeout=timeout, check=True,
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/opt/anaconda3/lib/python3.12/subprocess.py", line 548, in run
    with Popen(*popenargs, **kwargs) as process:
         ^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/opt/anaconda3/lib/python3.12/subprocess.py", line 1026, in __init__
    self._execute_child(args, executable, preexec_fn, close_fds,
  File "/opt/anaconda3/lib/python3.12/subprocess.py", line 1953, in _execute_child
    raise child_exception_type(errno_num, err_msg, err_filename)
FileNotFoundError: [Errno 2] No such file or directory: '/opt/anaconda3/lib/python3.12/site-packages/wandb/bin/apple_gpu_stats'
2024-10-06 21:14:23,765 DEBUG   HandlerThread:7667 [system_info.py:_save_conda():224] Saving conda packages done
2024-10-06 21:14:23,766 INFO    HandlerThread:7667 [system_monitor.py:probe():229] Finished publishing system info
2024-10-06 21:14:23,769 DEBUG   SenderThread:7667 [sender.py:send():379] send: files
2024-10-06 21:14:23,769 INFO    SenderThread:7667 [sender.py:_save_file():1454] saving file wandb-metadata.json with policy now
2024-10-06 21:14:24,018 DEBUG   HandlerThread:7667 [handler.py:handle_request():158] handle_request: python_packages
2024-10-06 21:14:24,018 DEBUG   SenderThread:7667 [sender.py:send_request():406] send_request: python_packages
2024-10-06 21:14:24,019 INFO    wandb-upload_0:7667 [upload_job.py:push():130] Uploaded file /var/folders/jv/wvjkll0d0vl_clbchr37hpk40000gn/T/tmpvj7olw2_wandb/95z5c85h-wandb-metadata.json
2024-10-06 21:14:24,019 DEBUG   HandlerThread:7667 [handler.py:handle_request():158] handle_request: stop_status
2024-10-06 21:14:24,019 DEBUG   HandlerThread:7667 [handler.py:handle_request():158] handle_request: internal_messages
2024-10-06 21:14:24,020 DEBUG   SenderThread:7667 [sender.py:send_request():406] send_request: stop_status
2024-10-06 21:14:24,071 DEBUG   SenderThread:7667 [sender.py:send():379] send: telemetry
2024-10-06 21:14:24,516 INFO    Thread-12 :7667 [dir_watcher.py:_on_file_modified():288] file/dir modified: /Users/milind/Desktop/valence/milind_code/wandb/run-20241006_211421-es5i1noe/files/conda-environment.yaml
2024-10-06 21:14:24,516 INFO    Thread-12 :7667 [dir_watcher.py:_on_file_created():271] file/dir created: /Users/milind/Desktop/valence/milind_code/wandb/run-20241006_211421-es5i1noe/files/wandb-metadata.json
2024-10-06 21:14:24,516 INFO    Thread-12 :7667 [dir_watcher.py:_on_file_created():271] file/dir created: /Users/milind/Desktop/valence/milind_code/wandb/run-20241006_211421-es5i1noe/files/requirements.txt
2024-10-06 21:14:25,046 DEBUG   HandlerThread:7667 [handler.py:handle_request():158] handle_request: internal_messages
2024-10-06 21:14:25,522 INFO    Thread-12 :7667 [dir_watcher.py:_on_file_created():271] file/dir created: /Users/milind/Desktop/valence/milind_code/wandb/run-20241006_211421-es5i1noe/files/output.log
2024-10-06 21:14:25,596 ERROR   gpuapple  :7667 [gpu_apple.py:sample():110] GPU stats error: [Errno 2] No such file or directory: '/opt/anaconda3/lib/python3.12/site-packages/wandb/bin/apple_gpu_stats'
Traceback (most recent call last):
  File "/opt/anaconda3/lib/python3.12/site-packages/wandb/sdk/internal/system/assets/gpu_apple.py", line 64, in sample
    subprocess.check_output(command, universal_newlines=True)
  File "/opt/anaconda3/lib/python3.12/subprocess.py", line 466, in check_output
    return run(*popenargs, stdout=PIPE, timeout=timeout, check=True,
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/opt/anaconda3/lib/python3.12/subprocess.py", line 548, in run
    with Popen(*popenargs, **kwargs) as process:
         ^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/opt/anaconda3/lib/python3.12/subprocess.py", line 1026, in __init__
    self._execute_child(args, executable, preexec_fn, close_fds,
  File "/opt/anaconda3/lib/python3.12/subprocess.py", line 1953, in _execute_child
    raise child_exception_type(errno_num, err_msg, err_filename)
FileNotFoundError: [Errno 2] No such file or directory: '/opt/anaconda3/lib/python3.12/site-packages/wandb/bin/apple_gpu_stats'
2024-10-06 21:14:26,039 DEBUG   HandlerThread:7667 [handler.py:handle_request():158] handle_request: internal_messages
2024-10-06 21:14:26,773 DEBUG   HandlerThread:7667 [handler.py:handle_request():158] handle_request: status_report
2024-10-06 21:14:27,039 DEBUG   HandlerThread:7667 [handler.py:handle_request():158] handle_request: internal_messages
2024-10-06 21:14:27,538 INFO    Thread-12 :7667 [dir_watcher.py:_on_file_modified():288] file/dir modified: /Users/milind/Desktop/valence/milind_code/wandb/run-20241006_211421-es5i1noe/files/output.log
2024-10-06 21:14:27,611 ERROR   gpuapple  :7667 [gpu_apple.py:sample():110] GPU stats error: [Errno 2] No such file or directory: '/opt/anaconda3/lib/python3.12/site-packages/wandb/bin/apple_gpu_stats'
Traceback (most recent call last):
  File "/opt/anaconda3/lib/python3.12/site-packages/wandb/sdk/internal/system/assets/gpu_apple.py", line 64, in sample
    subprocess.check_output(command, universal_newlines=True)
  File "/opt/anaconda3/lib/python3.12/subprocess.py", line 466, in check_output
    return run(*popenargs, stdout=PIPE, timeout=timeout, check=True,
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/opt/anaconda3/lib/python3.12/subprocess.py", line 548, in run
    with Popen(*popenargs, **kwargs) as process:
         ^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/opt/anaconda3/lib/python3.12/subprocess.py", line 1026, in __init__
    self._execute_child(args, executable, preexec_fn, close_fds,
  File "/opt/anaconda3/lib/python3.12/subprocess.py", line 1953, in _execute_child
    raise child_exception_type(errno_num, err_msg, err_filename)
FileNotFoundError: [Errno 2] No such file or directory: '/opt/anaconda3/lib/python3.12/site-packages/wandb/bin/apple_gpu_stats'
2024-10-06 21:14:28,041 DEBUG   HandlerThread:7667 [handler.py:handle_request():158] handle_request: internal_messages
2024-10-06 21:14:29,045 DEBUG   HandlerThread:7667 [handler.py:handle_request():158] handle_request: internal_messages
2024-10-06 21:14:29,627 ERROR   gpuapple  :7667 [gpu_apple.py:sample():110] GPU stats error: [Errno 2] No such file or directory: '/opt/anaconda3/lib/python3.12/site-packages/wandb/bin/apple_gpu_stats'
Traceback (most recent call last):
  File "/opt/anaconda3/lib/python3.12/site-packages/wandb/sdk/internal/system/assets/gpu_apple.py", line 64, in sample
    subprocess.check_output(command, universal_newlines=True)
  File "/opt/anaconda3/lib/python3.12/subprocess.py", line 466, in check_output
    return run(*popenargs, stdout=PIPE, timeout=timeout, check=True,
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/opt/anaconda3/lib/python3.12/subprocess.py", line 548, in run
    with Popen(*popenargs, **kwargs) as process:
         ^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/opt/anaconda3/lib/python3.12/subprocess.py", line 1026, in __init__
    self._execute_child(args, executable, preexec_fn, close_fds,
  File "/opt/anaconda3/lib/python3.12/subprocess.py", line 1953, in _execute_child
    raise child_exception_type(errno_num, err_msg, err_filename)
FileNotFoundError: [Errno 2] No such file or directory: '/opt/anaconda3/lib/python3.12/site-packages/wandb/bin/apple_gpu_stats'
2024-10-06 21:14:30,049 DEBUG   HandlerThread:7667 [handler.py:handle_request():158] handle_request: internal_messages
2024-10-06 21:14:31,064 DEBUG   HandlerThread:7667 [handler.py:handle_request():158] handle_request: internal_messages
2024-10-06 21:14:31,637 ERROR   gpuapple  :7667 [gpu_apple.py:sample():110] GPU stats error: [Errno 2] No such file or directory: '/opt/anaconda3/lib/python3.12/site-packages/wandb/bin/apple_gpu_stats'
Traceback (most recent call last):
  File "/opt/anaconda3/lib/python3.12/site-packages/wandb/sdk/internal/system/assets/gpu_apple.py", line 64, in sample
    subprocess.check_output(command, universal_newlines=True)
  File "/opt/anaconda3/lib/python3.12/subprocess.py", line 466, in check_output
    return run(*popenargs, stdout=PIPE, timeout=timeout, check=True,
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/opt/anaconda3/lib/python3.12/subprocess.py", line 548, in run
    with Popen(*popenargs, **kwargs) as process:
         ^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/opt/anaconda3/lib/python3.12/subprocess.py", line 1026, in __init__
    self._execute_child(args, executable, preexec_fn, close_fds,
  File "/opt/anaconda3/lib/python3.12/subprocess.py", line 1953, in _execute_child
    raise child_exception_type(errno_num, err_msg, err_filename)
FileNotFoundError: [Errno 2] No such file or directory: '/opt/anaconda3/lib/python3.12/site-packages/wandb/bin/apple_gpu_stats'
2024-10-06 21:14:32,034 DEBUG   HandlerThread:7667 [handler.py:handle_request():158] handle_request: status_report
2024-10-06 21:14:32,055 DEBUG   HandlerThread:7667 [handler.py:handle_request():158] handle_request: internal_messages
2024-10-06 21:14:33,060 DEBUG   HandlerThread:7667 [handler.py:handle_request():158] handle_request: internal_messages
2024-10-06 21:14:33,646 ERROR   gpuapple  :7667 [gpu_apple.py:sample():110] GPU stats error: [Errno 2] No such file or directory: '/opt/anaconda3/lib/python3.12/site-packages/wandb/bin/apple_gpu_stats'
Traceback (most recent call last):
  File "/opt/anaconda3/lib/python3.12/site-packages/wandb/sdk/internal/system/assets/gpu_apple.py", line 64, in sample
    subprocess.check_output(command, universal_newlines=True)
  File "/opt/anaconda3/lib/python3.12/subprocess.py", line 466, in check_output
    return run(*popenargs, stdout=PIPE, timeout=timeout, check=True,
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/opt/anaconda3/lib/python3.12/subprocess.py", line 548, in run
    with Popen(*popenargs, **kwargs) as process:
         ^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/opt/anaconda3/lib/python3.12/subprocess.py", line 1026, in __init__
    self._execute_child(args, executable, preexec_fn, close_fds,
  File "/opt/anaconda3/lib/python3.12/subprocess.py", line 1953, in _execute_child
    raise child_exception_type(errno_num, err_msg, err_filename)
FileNotFoundError: [Errno 2] No such file or directory: '/opt/anaconda3/lib/python3.12/site-packages/wandb/bin/apple_gpu_stats'
2024-10-06 21:14:34,064 DEBUG   HandlerThread:7667 [handler.py:handle_request():158] handle_request: internal_messages
2024-10-06 21:14:35,066 DEBUG   HandlerThread:7667 [handler.py:handle_request():158] handle_request: internal_messages
2024-10-06 21:14:35,650 ERROR   gpuapple  :7667 [gpu_apple.py:sample():110] GPU stats error: [Errno 2] No such file or directory: '/opt/anaconda3/lib/python3.12/site-packages/wandb/bin/apple_gpu_stats'
Traceback (most recent call last):
  File "/opt/anaconda3/lib/python3.12/site-packages/wandb/sdk/internal/system/assets/gpu_apple.py", line 64, in sample
    subprocess.check_output(command, universal_newlines=True)
  File "/opt/anaconda3/lib/python3.12/subprocess.py", line 466, in check_output
    return run(*popenargs, stdout=PIPE, timeout=timeout, check=True,
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/opt/anaconda3/lib/python3.12/subprocess.py", line 548, in run
    with Popen(*popenargs, **kwargs) as process:
         ^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/opt/anaconda3/lib/python3.12/subprocess.py", line 1026, in __init__
    self._execute_child(args, executable, preexec_fn, close_fds,
  File "/opt/anaconda3/lib/python3.12/subprocess.py", line 1953, in _execute_child
    raise child_exception_type(errno_num, err_msg, err_filename)
FileNotFoundError: [Errno 2] No such file or directory: '/opt/anaconda3/lib/python3.12/site-packages/wandb/bin/apple_gpu_stats'
2024-10-06 21:14:36,071 DEBUG   HandlerThread:7667 [handler.py:handle_request():158] handle_request: internal_messages
2024-10-06 21:14:37,073 DEBUG   HandlerThread:7667 [handler.py:handle_request():158] handle_request: internal_messages
2024-10-06 21:14:37,654 ERROR   gpuapple  :7667 [gpu_apple.py:sample():110] GPU stats error: [Errno 2] No such file or directory: '/opt/anaconda3/lib/python3.12/site-packages/wandb/bin/apple_gpu_stats'
Traceback (most recent call last):
  File "/opt/anaconda3/lib/python3.12/site-packages/wandb/sdk/internal/system/assets/gpu_apple.py", line 64, in sample
    subprocess.check_output(command, universal_newlines=True)
  File "/opt/anaconda3/lib/python3.12/subprocess.py", line 466, in check_output
    return run(*popenargs, stdout=PIPE, timeout=timeout, check=True,
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/opt/anaconda3/lib/python3.12/subprocess.py", line 548, in run
    with Popen(*popenargs, **kwargs) as process:
         ^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/opt/anaconda3/lib/python3.12/subprocess.py", line 1026, in __init__
    self._execute_child(args, executable, preexec_fn, close_fds,
  File "/opt/anaconda3/lib/python3.12/subprocess.py", line 1953, in _execute_child
    raise child_exception_type(errno_num, err_msg, err_filename)
FileNotFoundError: [Errno 2] No such file or directory: '/opt/anaconda3/lib/python3.12/site-packages/wandb/bin/apple_gpu_stats'
2024-10-06 21:14:37,769 DEBUG   HandlerThread:7667 [handler.py:handle_request():158] handle_request: status_report
2024-10-06 21:14:37,894 DEBUG   HandlerThread:7667 [handler.py:handle_request():158] handle_request: partial_history
2024-10-06 21:14:37,894 DEBUG   HandlerThread:7667 [handler.py:handle_request():158] handle_request: partial_history
2024-10-06 21:14:37,895 DEBUG   HandlerThread:7667 [handler.py:handle_request():158] handle_request: partial_history
2024-10-06 21:14:37,895 DEBUG   SenderThread:7667 [sender.py:send():379] send: history
2024-10-06 21:14:37,895 DEBUG   HandlerThread:7667 [handler.py:handle_request():158] handle_request: partial_history
2024-10-06 21:14:37,895 DEBUG   SenderThread:7667 [sender.py:send_request():406] send_request: summary_record
2024-10-06 21:14:37,895 DEBUG   HandlerThread:7667 [handler.py:handle_request():158] handle_request: partial_history
2024-10-06 21:14:37,896 DEBUG   HandlerThread:7667 [handler.py:handle_request():158] handle_request: partial_history
2024-10-06 21:14:37,897 DEBUG   HandlerThread:7667 [handler.py:handle_request():158] handle_request: partial_history
2024-10-06 21:14:37,897 DEBUG   HandlerThread:7667 [handler.py:handle_request():158] handle_request: partial_history
2024-10-06 21:14:37,897 DEBUG   HandlerThread:7667 [handler.py:handle_request():158] handle_request: partial_history
2024-10-06 21:14:37,897 DEBUG   HandlerThread:7667 [handler.py:handle_request():158] handle_request: partial_history
2024-10-06 21:14:37,897 DEBUG   HandlerThread:7667 [handler.py:handle_request():158] handle_request: partial_history
2024-10-06 21:14:37,898 INFO    SenderThread:7667 [sender.py:_save_file():1454] saving file wandb-summary.json with policy end
2024-10-06 21:14:37,898 DEBUG   SenderThread:7667 [sender.py:send():379] send: history
2024-10-06 21:14:37,898 DEBUG   SenderThread:7667 [sender.py:send_request():406] send_request: summary_record
2024-10-06 21:14:37,899 INFO    SenderThread:7667 [sender.py:_save_file():1454] saving file wandb-summary.json with policy end
2024-10-06 21:14:37,899 DEBUG   SenderThread:7667 [sender.py:send():379] send: history
2024-10-06 21:14:37,899 DEBUG   SenderThread:7667 [sender.py:send_request():406] send_request: summary_record
2024-10-06 21:14:37,899 INFO    SenderThread:7667 [sender.py:_save_file():1454] saving file wandb-summary.json with policy end
2024-10-06 21:14:37,899 DEBUG   SenderThread:7667 [sender.py:send():379] send: history
2024-10-06 21:14:37,899 DEBUG   SenderThread:7667 [sender.py:send_request():406] send_request: summary_record
2024-10-06 21:14:37,899 INFO    SenderThread:7667 [sender.py:_save_file():1454] saving file wandb-summary.json with policy end
2024-10-06 21:14:37,899 DEBUG   SenderThread:7667 [sender.py:send():379] send: history
2024-10-06 21:14:37,899 DEBUG   SenderThread:7667 [sender.py:send_request():406] send_request: summary_record
2024-10-06 21:14:37,900 INFO    SenderThread:7667 [sender.py:_save_file():1454] saving file wandb-summary.json with policy end
2024-10-06 21:14:37,900 DEBUG   SenderThread:7667 [sender.py:send():379] send: history
2024-10-06 21:14:37,900 DEBUG   SenderThread:7667 [sender.py:send_request():406] send_request: summary_record
2024-10-06 21:14:37,900 INFO    SenderThread:7667 [sender.py:_save_file():1454] saving file wandb-summary.json with policy end
2024-10-06 21:14:37,900 DEBUG   SenderThread:7667 [sender.py:send():379] send: history
2024-10-06 21:14:37,900 DEBUG   SenderThread:7667 [sender.py:send_request():406] send_request: summary_record
2024-10-06 21:14:37,900 INFO    SenderThread:7667 [sender.py:_save_file():1454] saving file wandb-summary.json with policy end
2024-10-06 21:14:37,900 DEBUG   SenderThread:7667 [sender.py:send():379] send: history
2024-10-06 21:14:37,900 DEBUG   SenderThread:7667 [sender.py:send_request():406] send_request: summary_record
2024-10-06 21:14:37,900 INFO    SenderThread:7667 [sender.py:_save_file():1454] saving file wandb-summary.json with policy end
2024-10-06 21:14:37,900 DEBUG   SenderThread:7667 [sender.py:send():379] send: history
2024-10-06 21:14:37,900 DEBUG   SenderThread:7667 [sender.py:send_request():406] send_request: summary_record
2024-10-06 21:14:37,900 INFO    SenderThread:7667 [sender.py:_save_file():1454] saving file wandb-summary.json with policy end
2024-10-06 21:14:37,900 DEBUG   SenderThread:7667 [sender.py:send():379] send: history
2024-10-06 21:14:37,901 DEBUG   SenderThread:7667 [sender.py:send_request():406] send_request: summary_record
2024-10-06 21:14:37,901 INFO    SenderThread:7667 [sender.py:_save_file():1454] saving file wandb-summary.json with policy end
2024-10-06 21:14:37,901 DEBUG   SenderThread:7667 [sender.py:send():379] send: history
2024-10-06 21:14:37,901 DEBUG   SenderThread:7667 [sender.py:send_request():406] send_request: summary_record
2024-10-06 21:14:37,901 INFO    SenderThread:7667 [sender.py:_save_file():1454] saving file wandb-summary.json with policy end
2024-10-06 21:14:38,078 DEBUG   HandlerThread:7667 [handler.py:handle_request():158] handle_request: internal_messages
2024-10-06 21:14:38,583 INFO    Thread-12 :7667 [dir_watcher.py:_on_file_created():271] file/dir created: /Users/milind/Desktop/valence/milind_code/wandb/run-20241006_211421-es5i1noe/files/wandb-summary.json
2024-10-06 21:14:39,084 DEBUG   HandlerThread:7667 [handler.py:handle_request():158] handle_request: stop_status
2024-10-06 21:14:39,084 DEBUG   SenderThread:7667 [sender.py:send_request():406] send_request: stop_status
2024-10-06 21:14:39,115 DEBUG   HandlerThread:7667 [handler.py:handle_request():158] handle_request: internal_messages
2024-10-06 21:14:39,547 DEBUG   SenderThread:7667 [sender.py:send():379] send: telemetry
2024-10-06 21:14:39,548 DEBUG   HandlerThread:7667 [handler.py:handle_request():158] handle_request: server_info
2024-10-06 21:14:39,548 DEBUG   SenderThread:7667 [sender.py:send_request():406] send_request: server_info
2024-10-06 21:14:39,589 INFO    Thread-12 :7667 [dir_watcher.py:_on_file_modified():288] file/dir modified: /Users/milind/Desktop/valence/milind_code/wandb/run-20241006_211421-es5i1noe/files/output.log
2024-10-06 21:14:39,589 DEBUG   SenderThread:7667 [sender.py:send():379] send: exit
2024-10-06 21:14:39,589 INFO    SenderThread:7667 [sender.py:send_exit():586] handling exit code: 0
2024-10-06 21:14:39,589 INFO    SenderThread:7667 [sender.py:send_exit():588] handling runtime: 18
2024-10-06 21:14:39,589 INFO    SenderThread:7667 [sender.py:_save_file():1454] saving file wandb-summary.json with policy end
2024-10-06 21:14:39,589 INFO    SenderThread:7667 [sender.py:send_exit():594] send defer
2024-10-06 21:14:39,589 DEBUG   HandlerThread:7667 [handler.py:handle_request():158] handle_request: defer
2024-10-06 21:14:39,589 INFO    HandlerThread:7667 [handler.py:handle_request_defer():184] handle defer: 0
2024-10-06 21:14:39,589 DEBUG   SenderThread:7667 [sender.py:send_request():406] send_request: defer
2024-10-06 21:14:39,589 INFO    SenderThread:7667 [sender.py:send_request_defer():610] handle sender defer: 0
2024-10-06 21:14:39,589 INFO    SenderThread:7667 [sender.py:transition_state():614] send defer: 1
2024-10-06 21:14:39,590 DEBUG   HandlerThread:7667 [handler.py:handle_request():158] handle_request: defer
2024-10-06 21:14:39,590 INFO    HandlerThread:7667 [handler.py:handle_request_defer():184] handle defer: 1
2024-10-06 21:14:39,590 DEBUG   SenderThread:7667 [sender.py:send_request():406] send_request: defer
2024-10-06 21:14:39,590 INFO    SenderThread:7667 [sender.py:send_request_defer():610] handle sender defer: 1
2024-10-06 21:14:39,590 INFO    SenderThread:7667 [sender.py:transition_state():614] send defer: 2
2024-10-06 21:14:39,590 DEBUG   HandlerThread:7667 [handler.py:handle_request():158] handle_request: defer
2024-10-06 21:14:39,590 INFO    HandlerThread:7667 [handler.py:handle_request_defer():184] handle defer: 2
2024-10-06 21:14:39,590 INFO    HandlerThread:7667 [system_monitor.py:finish():203] Stopping system monitor
2024-10-06 21:14:39,590 DEBUG   SystemMonitor:7667 [system_monitor.py:_start():172] Starting system metrics aggregation loop
2024-10-06 21:14:39,590 DEBUG   SystemMonitor:7667 [system_monitor.py:_start():179] Finished system metrics aggregation loop
2024-10-06 21:14:39,590 DEBUG   SystemMonitor:7667 [system_monitor.py:_start():183] Publishing last batch of metrics
2024-10-06 21:14:39,590 INFO    HandlerThread:7667 [interfaces.py:finish():200] Joined cpu monitor
2024-10-06 21:14:39,590 INFO    HandlerThread:7667 [interfaces.py:finish():200] Joined disk monitor
2024-10-06 21:14:39,590 INFO    HandlerThread:7667 [interfaces.py:finish():200] Joined gpuapple monitor
2024-10-06 21:14:39,590 INFO    HandlerThread:7667 [interfaces.py:finish():200] Joined memory monitor
2024-10-06 21:14:39,590 INFO    HandlerThread:7667 [interfaces.py:finish():200] Joined network monitor
2024-10-06 21:14:39,590 DEBUG   SenderThread:7667 [sender.py:send_request():406] send_request: defer
2024-10-06 21:14:39,590 INFO    SenderThread:7667 [sender.py:send_request_defer():610] handle sender defer: 2
2024-10-06 21:14:39,590 INFO    SenderThread:7667 [sender.py:transition_state():614] send defer: 3
2024-10-06 21:14:39,590 DEBUG   SenderThread:7667 [sender.py:send():379] send: telemetry
2024-10-06 21:14:39,590 DEBUG   HandlerThread:7667 [handler.py:handle_request():158] handle_request: defer
2024-10-06 21:14:39,590 INFO    HandlerThread:7667 [handler.py:handle_request_defer():184] handle defer: 3
2024-10-06 21:14:39,590 DEBUG   SenderThread:7667 [sender.py:send_request():406] send_request: defer
2024-10-06 21:14:39,590 INFO    SenderThread:7667 [sender.py:send_request_defer():610] handle sender defer: 3
2024-10-06 21:14:39,590 INFO    SenderThread:7667 [sender.py:transition_state():614] send defer: 4
2024-10-06 21:14:39,590 DEBUG   HandlerThread:7667 [handler.py:handle_request():158] handle_request: defer
2024-10-06 21:14:39,591 INFO    HandlerThread:7667 [handler.py:handle_request_defer():184] handle defer: 4
2024-10-06 21:14:39,591 DEBUG   SenderThread:7667 [sender.py:send_request():406] send_request: defer
2024-10-06 21:14:39,591 INFO    SenderThread:7667 [sender.py:send_request_defer():610] handle sender defer: 4
2024-10-06 21:14:39,591 INFO    SenderThread:7667 [sender.py:transition_state():614] send defer: 5
2024-10-06 21:14:39,591 DEBUG   HandlerThread:7667 [handler.py:handle_request():158] handle_request: defer
2024-10-06 21:14:39,591 INFO    HandlerThread:7667 [handler.py:handle_request_defer():184] handle defer: 5
2024-10-06 21:14:39,591 DEBUG   SenderThread:7667 [sender.py:send():379] send: summary
2024-10-06 21:14:39,591 INFO    SenderThread:7667 [sender.py:_save_file():1454] saving file wandb-summary.json with policy end
2024-10-06 21:14:39,591 DEBUG   SenderThread:7667 [sender.py:send_request():406] send_request: defer
2024-10-06 21:14:39,591 INFO    SenderThread:7667 [sender.py:send_request_defer():610] handle sender defer: 5
2024-10-06 21:14:39,591 INFO    SenderThread:7667 [sender.py:transition_state():614] send defer: 6
2024-10-06 21:14:39,591 DEBUG   HandlerThread:7667 [handler.py:handle_request():158] handle_request: defer
2024-10-06 21:14:39,591 INFO    HandlerThread:7667 [handler.py:handle_request_defer():184] handle defer: 6
2024-10-06 21:14:39,591 DEBUG   SenderThread:7667 [sender.py:send_request():406] send_request: defer
2024-10-06 21:14:39,591 INFO    SenderThread:7667 [sender.py:send_request_defer():610] handle sender defer: 6
2024-10-06 21:14:39,592 DEBUG   HandlerThread:7667 [handler.py:handle_request():158] handle_request: status_report
2024-10-06 21:14:39,660 INFO    SenderThread:7667 [sender.py:transition_state():614] send defer: 7
2024-10-06 21:14:39,660 DEBUG   HandlerThread:7667 [handler.py:handle_request():158] handle_request: defer
2024-10-06 21:14:39,660 INFO    HandlerThread:7667 [handler.py:handle_request_defer():184] handle defer: 7
2024-10-06 21:14:39,660 DEBUG   SenderThread:7667 [sender.py:send_request():406] send_request: defer
2024-10-06 21:14:39,660 INFO    SenderThread:7667 [sender.py:send_request_defer():610] handle sender defer: 7
2024-10-06 21:14:40,595 INFO    Thread-12 :7667 [dir_watcher.py:_on_file_modified():288] file/dir modified: /Users/milind/Desktop/valence/milind_code/wandb/run-20241006_211421-es5i1noe/files/config.yaml
2024-10-06 21:14:40,596 DEBUG   HandlerThread:7667 [handler.py:handle_request():158] handle_request: poll_exit
2024-10-06 21:14:40,596 INFO    Thread-12 :7667 [dir_watcher.py:_on_file_modified():288] file/dir modified: /Users/milind/Desktop/valence/milind_code/wandb/run-20241006_211421-es5i1noe/files/wandb-summary.json
2024-10-06 21:14:41,599 INFO    Thread-12 :7667 [dir_watcher.py:_on_file_modified():288] file/dir modified: /Users/milind/Desktop/valence/milind_code/wandb/run-20241006_211421-es5i1noe/files/output.log
2024-10-06 21:14:42,778 INFO    SenderThread:7667 [sender.py:transition_state():614] send defer: 8
2024-10-06 21:14:42,779 DEBUG   SenderThread:7667 [sender.py:send_request():406] send_request: poll_exit
2024-10-06 21:14:42,779 DEBUG   HandlerThread:7667 [handler.py:handle_request():158] handle_request: defer
2024-10-06 21:14:42,780 INFO    HandlerThread:7667 [handler.py:handle_request_defer():184] handle defer: 8
2024-10-06 21:14:42,780 DEBUG   SenderThread:7667 [sender.py:send_request():406] send_request: defer
2024-10-06 21:14:42,780 INFO    SenderThread:7667 [sender.py:send_request_defer():610] handle sender defer: 8
2024-10-06 21:14:42,780 INFO    SenderThread:7667 [job_builder.py:build():440] Attempting to build job artifact
2024-10-06 21:14:42,781 INFO    SenderThread:7667 [job_builder.py:_get_source_type():580] no source found
2024-10-06 21:14:42,781 INFO    SenderThread:7667 [sender.py:transition_state():614] send defer: 9
2024-10-06 21:14:42,782 DEBUG   HandlerThread:7667 [handler.py:handle_request():158] handle_request: defer
2024-10-06 21:14:42,782 INFO    HandlerThread:7667 [handler.py:handle_request_defer():184] handle defer: 9
2024-10-06 21:14:42,782 DEBUG   SenderThread:7667 [sender.py:send_request():406] send_request: defer
2024-10-06 21:14:42,782 INFO    SenderThread:7667 [sender.py:send_request_defer():610] handle sender defer: 9
2024-10-06 21:14:42,782 INFO    SenderThread:7667 [dir_watcher.py:finish():358] shutting down directory watcher
2024-10-06 21:14:43,606 INFO    SenderThread:7667 [dir_watcher.py:_on_file_modified():288] file/dir modified: /Users/milind/Desktop/valence/milind_code/wandb/run-20241006_211421-es5i1noe/files/output.log
2024-10-06 21:14:43,607 INFO    SenderThread:7667 [dir_watcher.py:finish():388] scan: /Users/milind/Desktop/valence/milind_code/wandb/run-20241006_211421-es5i1noe/files
2024-10-06 21:14:43,607 INFO    SenderThread:7667 [dir_watcher.py:finish():402] scan save: /Users/milind/Desktop/valence/milind_code/wandb/run-20241006_211421-es5i1noe/files/requirements.txt requirements.txt
2024-10-06 21:14:43,608 INFO    SenderThread:7667 [dir_watcher.py:finish():402] scan save: /Users/milind/Desktop/valence/milind_code/wandb/run-20241006_211421-es5i1noe/files/output.log output.log
2024-10-06 21:14:43,608 INFO    SenderThread:7667 [dir_watcher.py:finish():402] scan save: /Users/milind/Desktop/valence/milind_code/wandb/run-20241006_211421-es5i1noe/files/config.yaml config.yaml
2024-10-06 21:14:43,610 DEBUG   HandlerThread:7667 [handler.py:handle_request():158] handle_request: poll_exit
2024-10-06 21:14:43,612 INFO    SenderThread:7667 [dir_watcher.py:finish():402] scan save: /Users/milind/Desktop/valence/milind_code/wandb/run-20241006_211421-es5i1noe/files/wandb-summary.json wandb-summary.json
2024-10-06 21:14:43,614 INFO    SenderThread:7667 [dir_watcher.py:finish():402] scan save: /Users/milind/Desktop/valence/milind_code/wandb/run-20241006_211421-es5i1noe/files/conda-environment.yaml conda-environment.yaml
2024-10-06 21:14:43,615 INFO    SenderThread:7667 [dir_watcher.py:finish():402] scan save: /Users/milind/Desktop/valence/milind_code/wandb/run-20241006_211421-es5i1noe/files/wandb-metadata.json wandb-metadata.json
2024-10-06 21:14:43,615 INFO    SenderThread:7667 [sender.py:transition_state():614] send defer: 10
2024-10-06 21:14:43,615 DEBUG   SenderThread:7667 [sender.py:send_request():406] send_request: poll_exit
2024-10-06 21:14:43,616 DEBUG   HandlerThread:7667 [handler.py:handle_request():158] handle_request: defer
2024-10-06 21:14:43,616 INFO    HandlerThread:7667 [handler.py:handle_request_defer():184] handle defer: 10
2024-10-06 21:14:43,619 DEBUG   SenderThread:7667 [sender.py:send_request():406] send_request: defer
2024-10-06 21:14:43,620 INFO    SenderThread:7667 [sender.py:send_request_defer():610] handle sender defer: 10
2024-10-06 21:14:43,620 INFO    SenderThread:7667 [file_pusher.py:finish():169] shutting down file pusher
2024-10-06 21:14:43,804 INFO    wandb-upload_0:7667 [upload_job.py:push():130] Uploaded file /Users/milind/Desktop/valence/milind_code/wandb/run-20241006_211421-es5i1noe/files/requirements.txt
2024-10-06 21:14:43,841 INFO    wandb-upload_3:7667 [upload_job.py:push():130] Uploaded file /Users/milind/Desktop/valence/milind_code/wandb/run-20241006_211421-es5i1noe/files/wandb-summary.json
2024-10-06 21:14:43,851 INFO    wandb-upload_1:7667 [upload_job.py:push():130] Uploaded file /Users/milind/Desktop/valence/milind_code/wandb/run-20241006_211421-es5i1noe/files/output.log
2024-10-06 21:14:43,862 INFO    wandb-upload_2:7667 [upload_job.py:push():130] Uploaded file /Users/milind/Desktop/valence/milind_code/wandb/run-20241006_211421-es5i1noe/files/config.yaml
2024-10-06 21:14:43,864 INFO    wandb-upload_4:7667 [upload_job.py:push():130] Uploaded file /Users/milind/Desktop/valence/milind_code/wandb/run-20241006_211421-es5i1noe/files/conda-environment.yaml
2024-10-06 21:14:44,069 INFO    Thread-11 (_thread_body):7667 [sender.py:transition_state():614] send defer: 11
2024-10-06 21:14:44,070 DEBUG   HandlerThread:7667 [handler.py:handle_request():158] handle_request: defer
2024-10-06 21:14:44,071 INFO    HandlerThread:7667 [handler.py:handle_request_defer():184] handle defer: 11
2024-10-06 21:14:44,071 DEBUG   SenderThread:7667 [sender.py:send_request():406] send_request: defer
2024-10-06 21:14:44,071 INFO    SenderThread:7667 [sender.py:send_request_defer():610] handle sender defer: 11
2024-10-06 21:14:44,071 INFO    SenderThread:7667 [file_pusher.py:join():175] waiting for file pusher
2024-10-06 21:14:44,072 INFO    SenderThread:7667 [sender.py:transition_state():614] send defer: 12
2024-10-06 21:14:44,073 DEBUG   HandlerThread:7667 [handler.py:handle_request():158] handle_request: defer
2024-10-06 21:14:44,073 INFO    HandlerThread:7667 [handler.py:handle_request_defer():184] handle defer: 12
2024-10-06 21:14:44,073 DEBUG   SenderThread:7667 [sender.py:send_request():406] send_request: defer
2024-10-06 21:14:44,073 INFO    SenderThread:7667 [sender.py:send_request_defer():610] handle sender defer: 12
2024-10-06 21:14:44,073 INFO    SenderThread:7667 [file_stream.py:finish():601] file stream finish called
2024-10-06 21:14:44,174 INFO    SenderThread:7667 [file_stream.py:finish():605] file stream finish is done
2024-10-06 21:14:44,174 INFO    SenderThread:7667 [sender.py:transition_state():614] send defer: 13
2024-10-06 21:14:44,175 DEBUG   HandlerThread:7667 [handler.py:handle_request():158] handle_request: defer
2024-10-06 21:14:44,175 INFO    HandlerThread:7667 [handler.py:handle_request_defer():184] handle defer: 13
2024-10-06 21:14:44,175 DEBUG   SenderThread:7667 [sender.py:send_request():406] send_request: defer
2024-10-06 21:14:44,175 INFO    SenderThread:7667 [sender.py:send_request_defer():610] handle sender defer: 13
2024-10-06 21:14:44,176 INFO    SenderThread:7667 [sender.py:transition_state():614] send defer: 14
2024-10-06 21:14:44,176 DEBUG   HandlerThread:7667 [handler.py:handle_request():158] handle_request: defer
2024-10-06 21:14:44,176 DEBUG   SenderThread:7667 [sender.py:send():379] send: final
2024-10-06 21:14:44,176 INFO    HandlerThread:7667 [handler.py:handle_request_defer():184] handle defer: 14
2024-10-06 21:14:44,176 DEBUG   SenderThread:7667 [sender.py:send():379] send: footer
2024-10-06 21:14:44,176 DEBUG   SenderThread:7667 [sender.py:send_request():406] send_request: defer
2024-10-06 21:14:44,176 INFO    SenderThread:7667 [sender.py:send_request_defer():610] handle sender defer: 14
2024-10-06 21:14:44,178 DEBUG   HandlerThread:7667 [handler.py:handle_request():158] handle_request: poll_exit
2024-10-06 21:14:44,178 DEBUG   SenderThread:7667 [sender.py:send_request():406] send_request: poll_exit
2024-10-06 21:14:44,179 DEBUG   HandlerThread:7667 [handler.py:handle_request():158] handle_request: internal_messages
2024-10-06 21:14:44,180 DEBUG   HandlerThread:7667 [handler.py:handle_request():158] handle_request: get_summary
2024-10-06 21:14:44,180 DEBUG   HandlerThread:7667 [handler.py:handle_request():158] handle_request: sampled_history
2024-10-06 21:14:44,182 DEBUG   HandlerThread:7667 [handler.py:handle_request():158] handle_request: shutdown
2024-10-06 21:14:44,182 INFO    HandlerThread:7667 [handler.py:finish():882] shutting down handler
2024-10-06 21:14:45,181 INFO    SenderThread:7667 [sender.py:finish():1608] shutting down sender
2024-10-06 21:14:45,181 INFO    WriterThread:7667 [datastore.py:close():296] close: /Users/milind/Desktop/valence/milind_code/wandb/run-20241006_211421-es5i1noe/run-es5i1noe.wandb
2024-10-06 21:14:45,181 INFO    SenderThread:7667 [file_pusher.py:finish():169] shutting down file pusher
2024-10-06 21:14:45,181 INFO    SenderThread:7667 [file_pusher.py:join():175] waiting for file pusher
